common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2020,11 +2020,23 @@
     ],
     "sqlState" : "XX000"
   },
-  "INTERVAL_ARITHMETIC_OVERFLOW" : {
-    "message" : [
-      "<message>.<alternative>"
+  "INTERVAL_ARITHMETIC_OVERFLOW": {
+    "message": [
+      "Integer overflow while operating with intervals."
     ],
-    "sqlState" : "22015"
+    "subClass": {
+      "WITH_TRY_SUGGESTION": {
+        "message": [
+          "<message>.<alternative>."
+        ]
+      },
+      "WITHOUT_SUGGESTION": {
+        "message": [
+          ""
```

Comment:
We can say something like Try devising appropriate values for the interval parameters. Or something similar.